tools/clang/test/CodeGenSPIRV_Lit/intrinsics.get-attribute-at-vertex.s.funcParam.hlsl

```diff
@@ -0,0 +1,21 @@
+// RUN: %dxc -T ps_6_1 -E main  %s -spirv -Od 2>&1 | FileCheck %s
```

Comment:
If you use `-Od` the legalization pass could still run. Using `-fcgl` will make sure the legalization passes do not run either.

```suggestion
// RUN: %dxc -T ps_6_1 -E main  %s -spirv -fcgl 2>&1 | FileCheck %s
```

Reply:
Will fix.